spec/features/create_work_spec.rb

```diff
@@ -55,7 +56,7 @@
 
     select 'Attribution-ShareAlike 4.0 International', from: "#{work_type}_rights"
     choose("#{work_type}_visibility_open")
-    expect(page).to have_content('Please note, making something visible to the world (i.e. marking this as Public) may be viewed as publishing which could impact your ability to')
+    #expect(page).to have_content('Please note, making something visible to the world (i.e. marking this as Public) may be viewed as publishing which could impact your ability to')
```

Comment:
@hortongn Yeah - I meant to make a note about that - this is causing it to fail in an unhelpful way because we don't have the locale set up to display the help text in the sidebar. Do we have an issue for fixing that?